app/views/provider_user_mailer/activation_reminder.text.erb

```diff
@@ -1,10 +1,11 @@
 <%= @user.informal_name %>,
 
-A couple of days ago you signed up for <%= master_mailer_name %> to manage your API.
```

Comment:
Hmm. I think `master_mailer_name` was here to support for on-premises as well. So it's not always "Red Hat 3scale":

https://github.com/3scale/porta/blob/53d21a83f6f341111f4cd3cffac514847ef00a02/app/helpers/mailers_helper.rb#L4-L6

Reply:
@gui 
https://github.com/3scale/porta/pull/585#discussion_r254615032